packages/browser/src/core/events/__tests__/index.test.ts

```diff
@@ -156,6 +156,14 @@ describe('Event Factory', () => {
       expect(track.context).toEqual({ opt1: true })
     })
 
+    test('sets context correctly if property arg is undefined', () => {
```

Comment:
The bug didn't make it this far per se, but I wanted to harden this test suite again as it was a possible pain point.